src/assets/styles/app.css

```diff
@@ -959,6 +963,27 @@ aside h2 {
     margin-block-start: 5rem;
 }
 
+@media all and (max-width: 24rem) {
+    .site-brand {
+        margin-block: clamp(0.5rem, (2 - var(--fl-textSize-factor, 1)) * 1.28125rem, 1.28125rem);
+    }
+
+    .site-header .wrapper {
+        padding-left: 10px;
+        padding-right: 10px;
+    }
+
+    .site-nav {
+        margin: clamp(0, (2 - var(--fl-textSize-factor, 1)) * 0.5rem, 0.5rem);
```

Comment:
If the number `2` in this formula means `2rem`, it would be useful to write `2rem` in the formula for clarification.

Moreover, `(2 - var(--fl-textSize-factor, 1)) * 0.5rem` makes sense when the range of (min, max) is (1, 2) as the calculated value scales `--fl-textSize-factor` from (2, 1) into this (min, max). But it doesn't seem reasonable with other ranges such as (0, 0.5) in this case. because the calculated value will never fall into this range so the max value will always be applied. I'd love to hear your thoughts if I misunderstood.

[This thread](https://stackoverflow.com/questions/5294955/how-to-scale-down-a-range-of-numbers-with-a-known-min-and-max-value) talks about converting a range of numbers into another range, which might be helpful.

This calculation worths a code comment to explain its intention. Thanks.

Reply:
I issued https://github.com/BlueSlug/floeproject.org/pull/1 against your current branch to fix the overflow of the nav bar on mobile devices. Test results across a few mobile views look good but I'm not sure if I missed any side effects. @BlueSlug, can you please double check? Thanks.

Reply:
@cindyli it works quite well for the most part! I didn't notice any issues with phone emulation or while testing on my actual device. The only issues I noticed occurred when I was using the "Responsive" viewport emulation in Chrome and shrunk the width down to some fairly narrow widths.
<img width="344" alt="image" src="https://user-images.githubusercontent.com/547933/149396675-3c8b9fb9-28e6-4d46-abb2-f4440fa8ebdd.png">

I reintroduced my narrow-width media query rules as well as a couple of rules I was working on yesterday, though overflow is still occurring at even smaller widths (about 30px narrower). If this is acceptable, I will implement the removal of margin rules on the `.disclosure-button` class from your PR, but will keep the small-width media query and my new rules.
<img width="344" alt="image" src="https://user-images.githubusercontent.com/547933/149397208-918e0639-adb6-4517-9c02-6e1ac4d3219d.png">

Reply:
> If the number `2` in this formula means `2rem`, it would be useful to write `2rem` in the formula for clarification.
> 
> Moreover, `(2 - var(--fl-textSize-factor, 1)) * 0.5rem` makes sense when the range of (min, max) is (1, 2) as the calculated value scales `--fl-textSize-factor` from (2, 1) into this (min, max). But it doesn't seem reasonable with other ranges such as (0, 0.5) in this case. because the calculated value will never fall into this range so the max value will always be applied. I'd love to hear your thoughts if I misunderstood.
> 
> [This thread](https://stackoverflow.com/questions/5294955/how-to-scale-down-a-range-of-numbers-with-a-known-min-and-max-value) talks about converting a range of numbers into another range, which might be helpful.
> 
> This calculation worths a code comment to explain its intention. Thanks.

As we discussed earlier today but for the benefit of anyone else reading this, the `2` represents the maximum value of the `--fl-textSize-factor` variable, which is a unitless factor being applied globally to all text on the page. I can add a comment explaining how this calculation works :)

Re the behaviour at smaller factor values:
- at text-size == 1, the CSS custom property isn't defined, so the fallback of 1 is used and the maximum margin is applied
- at text-size < 1, the calculation will result in a size greater than `0.4rem`, however the `clamp()` function will prevent it from increasing past this maximum, as you've noted. This is by design and means the margins are unchanged when the UIO text-size preference is less than 1. Perhaps I've misunderstood your comment, though, so likewise I'd love to know if that's the case! :)

Reply:
> If this is acceptable, I will implement the removal of margin rules on the .disclosure-button class from your PR, but will keep the small-width media query and my new rules.

As tested and discussed yesterday, adding the small-width media query only fixes the overlapping issue on the narrow-width screens by 30 more pixels. After that, the overlapping issue will occur again. Considering the code complexity introduced for this small improvement, the decision is not to add the small-width media query section.

Moreover, as the overlapping issue doesn't occur on any device emulator but only on the "Responsive" viewport emulation for fairly narrow widths, we decided to go ahead with the solution of removing margin rules on the .disclosure-button class.